docs/source/troubleshooting/troubleshooting-misc.adoc

```diff
@@ -59,4 +59,35 @@ For example:
 
 ----
 minishift start --iso-url centos
-----
\ No newline at end of file
+----
+
+[[x509-certificate-is-valid-for-10.0.2.15-127.0.0.1-192.168.99.100-not-192.168.99.101]]
```

Comment:
I'm not sure if we can do anything about this, but this anchor name is unwieldy.

Reply:
@praveenkumar I would suggest the anchor name as `invalid-x509-certificate`. The anchor is used for pointing to the exact section e.g. `external-network-check` is the anchor in https://docs.openshift.org/latest/minishift/troubleshooting/troubleshooting-getting-started.html#external-network-check

Reply:
Agree, with @LalatenduMohanty  to have general anchor name.

Reply:
@LalatenduMohanty @budhrg I thought anchor name should be something which easily appears when user search for an error and an error message would be good as an anchor. I defer it to @robin-owen view.

Reply:
I also agree with @LalatenduMohanty and @budhrg here. We want to keep the title the same, just change the anchor name itself.